acceptance/go.mod

```diff
@@ -10,35 +10,43 @@ require (
 	github.com/hashicorp/go-uuid v1.0.3
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/vault/api v1.2.0
-	github.com/stretchr/testify v1.7.2
+	github.com/stretchr/testify v1.8.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
+	k8s.io/api v0.26.3
+	k8s.io/apimachinery v0.26.3
+	k8s.io/client-go v0.26.3
+	sigs.k8s.io/controller-runtime v0.14.6
+	sigs.k8s.io/gateway-api v0.7.0
```

Comment:
these changes were necessary for compatibility between the api spec (which has a dep on the runtime version) and the kubernetes libraries (which the runtime is incompatible with for newer versions). This should be fine since this now matches what's in the control-plane portion of the repo.